### PR TITLE
DEP-879: Changed times for 21/09/2020

### PR DESCRIPTION
**Pull request by tonymyers**

### CODE REVIEW

File: app/views/EmploymentRelatedSecuritiesView.scala.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
     appConfig: AppConfig)
 
 @()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
+non-UK-resident-landlords
```
Comment (Burdis) [inline app/views/EmploymentRelatedSecuritiesView.scala.html:24]: Does this need to be added?

File: app/views/TaxCreditsView.scala.html [hunk 2]
```diff
@@ -32,9 +32,8 @@ <h1 @if(!isIvrRedirect){class="header-margin-top"}>@messages("tax.credits.title"
 
     <div class="govuk-inset-text">
         <p>@messages("global.opening.times.title")</p>
-        <p>@messages("global.opening.times.mon.sat.8to10")</p>
-        <p>@messages("global.opening.times.sun.9to10")</p>
-        <p>@messages("global.closed.time.bankholidays")</p>
+        <p>@messages("global.opening.times.mon.sat.8to8")</p>
+        <p>@messages("global.closed.time.sun.banckholidays")</p>
```
Comment (Burdis) [inline app/views/TaxCreditsView.scala.html:36]: should be `bankholidays`

File: app/views/EmploymentRelatedSecuritiesView.scala.html (hunk 1 above)
Comment (tonymyers) [reply app/views/EmploymentRelatedSecuritiesView.scala.html:24]: No, it doesn't, well spotted. I'll remove

File: app/views/TaxCreditsView.scala.html (hunk 2 above)
Comment (tonymyers) [reply app/views/TaxCreditsView.scala.html:36]: Ticket states 

Opening times:
Monday to Saturday, 8am to 8pm
Closed Sundays and bank holidays.

So this is correct

Comment (Burdis) [reply app/views/TaxCreditsView.scala.html:36]: sorry mate you are correct I meant to change the spelling of banckholidays @tonymyers

**Review by Burdis — APPROVED** [review]

**Pull request merged**